requirements.txt

```diff
@@ -81,3 +81,4 @@ tablib==0.11.3
 Unidecode==0.04.19
 wsgiref==0.1.2
 YURL==0.13
+django-summernote==0.6.1
```

Comment:
In teoria si può usare il widget `TextEditorWidget` di `djangocms-text-ckeditor` (che è già installato), ma credo non valga la pena. se hai già fatto un giro direi che va bene così